README.md

```diff
@@ -52,7 +52,7 @@ We're back! Use this repo to share and keep track of software, tech, CS, PM, qua
  - 🇺🇸 - Requires U.S. Citizenship
  - 🔒 - Internship application is closed
 
-> For off-season internships, please see the [Off-Season 2024 README](./README-Off-Season.md).
+> For off-season internships, please see the [Off-Season 2024 README](https://github.com/SimplifyJobs/Summer2024-Internships/blob/dev/README-Off-Season.md).
```

Comment:
I think changing it to `(README-Off-Season.md)`  should work :thinking:

Reply:
Strange, this used to work: https://github.blog/2013-01-31-relative-links-in-markup-files/

Maybe something is broken on github's side